src/valhalla_run_route.cc

```diff
@@ -676,8 +676,6 @@ int main(int argc, char *argv[]) {
     PathAlgorithm* pathalgorithm;
     if (routetype == "multimodal") {
       pathalgorithm = &mm;
-    } else if (routetype == "pedestrian") {
-      pathalgorithm = &bd;
```

Comment:
this is inline with what the service does as well. once we refactor these test programs to use the `worker` methods we'll not need to worry about this logic.